### ISSUE TITLE
Blight + QtSixA/Ubuntu = No play

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#394]
Originally reported on Google Code with ID 395

```
Describe your system:
 - Operating System (be specific): Ubuntu 10.10
 - Machine type (32-bit or 64-bit):64-Bit
 - Mupen64Plus version: 1.5 from Ubuntu Software Center
 - Plugins used:Blight SDL input plugin 0.0.10

Describe the problem: When my PS3 Controller is connected my game won't start. Mupen
starts and allows me to configure Blight and set my bindings but when I start to play
any rom it freezes. However the Moment I disconnect my PS3 controller the game loads.


Please provide any additional information below.
The program I use to connect my PS3 Controller is here:
http://ubuntuforums.org/showthread.php?p=10069468
And I posted my problem with terminal output here:
http://ubuntuforums.org/showthread.php?p=10069468#post10069468

```

Reported by `phillip.griego` on 2010-11-04 05:04:16

**Comment by richard42** [comment]
```
Never mind.
Using QtSixA the ps3 controller works perfect via USB
for bluetooth you have to disable the rumble feature in QtSixA

There is no error with Mupen64Plus
Please close as invalid
```

Reported by `phillip.griego` on 2010-11-04 05:32:17

**Comment by richard42** [comment]
```
Thanks for feedback ;)
```

Reported by `dorian.fevrier` on 2010-11-29 22:58:56
- **Status changed**: `Invalid`

**Comment by richard42** [comment]
```
Hi, QtSixA dev here.

This was caused by a bad implementation of the rumble effect.
Now fixed in the PPA (v1.5.0 Beta)

(make sure you enable 'new rumble mode')
```

Reported by `falktx` on 2010-12-04 03:16:49

**Comment by richard42** [comment]
```
Thanks falktx! :)
```

Reported by `dorian.fevrier` on 2010-12-05 00:42:23
- **Status changed**: `External`
